docs/hooks/useScaffoldWriteContract.md

```diff
@@ -10,7 +10,15 @@ Use this hook to send a transaction to your smart contract to write data or perf
 const { writeContractAsync: writeYourContractAsync } = useScaffoldWriteContract({ contractName: "YourContract" });
 ```
 
-The first argument is name of the contract to write to and the second argument is wagmi's `useWriteContract` hook [parameters object](https://wagmi.sh/react/api/hooks/useWriteContract#parameters).
+Following configuration can be passed to hook:
```

Comment:
Nitpick, ignore if too formal:

`The following configuration options can be passed to the hook:`